docs/android-startup-scenarios.md

```diff
@@ -0,0 +1,62 @@
+
+# Android Startup Test
+
+## Prereqs
+
+- Ensure `python` is installed and available. Any currently supported `python` 3.* version should work. Downloads are available at https://www.python.org/downloads/.
+- Ensure `dotnet` is installed and available with the `dotnet` command for easy xharness installation. Any supported .NET Core version should work. [Dotnet Download](https://dotnet.microsoft.com/en-us/download) or [Daily Dotnet Download](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md).
+- Ensure `xharness` is installed and available with the `xharness` command. The current version in use can be found in the `eng/performance/maui_scenarios_android.proj` file at line 7 (under the tag `MicrosoftDotNetXHarnessCLIVersion`), although any recent version should work. [XHarness Install Instructions](https://github.com/dotnet/xharness?tab=readme-ov-file#installation-and-usage).
+- Have an Android app APK available for testing.
+- Have an Android Device (with developer mode enabled) or emulator connected to computer, and viewable with `xharness android device` or `xharness android adb -- devices -l`.
+
+## Steps
+
+1. Initialize the environment (note the . for bash):
+
+    ```sh
+    cd src/scenarios
+    . ./init.sh  # or `.\init.ps1` on Windows. Can specify custom dotnet install with -dotnetdir <dir>, but dotnet install should not impact Android Startup testing itself.
+    ```
+
+2. Navigate to the `genericandroidstartup` scenario directory:
+
+    ```sh
+    cd genericandroidstartup
+    ```
+
+3. Copy the APK into the `genericandroidstartup` directory.
+4. Run the test:
+
+    ```sh
+    python/(py -3 on Linux) test.py devicestartup --device-type android --package-path <path to apk (e.g. .)>/<apkname>.apk --package-name <apk package name> [--disable-animations] [--use-fully-drawn-time --fully-drawn-extra-delay <delay in sec> (**see below for note**)]
+    ```
```

Comment:
This looks cryptic when observed as a shell command. I suggest splitting it into multiple commands for example:

- non-Linux

```sh
python test.py devicestartup --device-type android --package-path <path-to-apk> --package-name <apk-package name> [--disable-animations] [--use-fully-drawn-time --fully-drawn-extra-delay <delay-in-sec>]
``` 

- Linux

```sh
py -3 test.py devicestartup --device-type android --package-path <path-to-apk> --package-name <apk-package name> [--disable-animations] [--use-fully-drawn-time --fully-drawn-extra-delay <delay-in-sec>]
``` 

- Please refer to the [Notes](./android-startup-scenarios#notes) below about specifying `--use-fully-drawn-time --fully-drawn-extra-delay` parameters

Reply:
Seems we can use python for both windows and Linux, so I kept just 1 reference. Also added the reference to the notes separately.